docs/Spark/gems/transform/data-cleansing.md

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 10
+title: Data Cleansing
+id: data-cleansing
+description: Standardize data formats and address missing or null values in the data.
+tags:
+  - gems
+  - clean
+  - format
+---
+
+Use the Data Cleansing Gem to standardize data formats and address missing or null values in the data.
+
+## Parameters
+| Parameter                        | Description                                                                                              | Required |
+|----------------------------------|----------------------------------------------------------------------------------------------------------|----------|
+| Select columns you want to clean | The set of columns on which to perform cleaning transformations. This is also the output set of columns. | False    |
+| Remove null data                 | The method used to remove null data.                                                                     | True     |
```

Comment:
Technically this is not required (doesn't throw an error), but it is automatically populated.